astropy/time/formats.py

```diff
@@ -2086,6 +2098,9 @@ class TimeDeltaFormat(TimeFormat):
     """Base class for time delta representations."""
 
     _registry = TIME_DELTA_FORMATS
+    _default_precision = 3
+    _min_precision = -99
```

Comment:
Agreed the 99 is weird and I struggled a bit there. For `Time` there is a real limit to precision because that gets passed into ERFA routines. For `TimeDelta` there is not really a limit. Currently for `TimeDelta` formats the precision only gets used via `np.round()` in `quantity_str`.

There are other options like overriding the precision setter or allowing `None` as a value for `_min/max_precision`, but I think keeps the code simple.

Reply:
OK, let's keep it as is.